taxi/forms.py

```diff
@@ -39,14 +52,34 @@ def clean_license_number(self):
         return validate_license_number(self.cleaned_data["license_number"])
 
 
-def validate_license_number(
-    license_number,
-):  # regex validation is also possible here
-    if len(license_number) != 8:
-        raise ValidationError("License number should consist of 8 characters")
-    elif not license_number[:3].isupper() or not license_number[:3].isalpha():
-        raise ValidationError("First 3 characters should be uppercase letters")
-    elif not license_number[3:].isdigit():
-        raise ValidationError("Last 5 characters should be digits")
+class DriverSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=55,
+        required=False,
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Search by username"}),
+    )
 
-    return license_number
+
+class CarSearchForm(forms.Form):
+    model = forms.CharField(
+        max_length=55,
+        required=False,
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Search by model"}),
+    )
+
+
+class ManufacturerSearchForm(forms.Form):
+    name = forms.CharField(
+        max_length=55,
+        required=True,
```

Comment:
The 'required' attribute for the 'name' field in 'ManufacturerSearchForm' is set to True. This means the search form will not allow empty searches, which might not be the intended behavior for a search form. Consider setting 'required' to False to allow users to perform searches without entering a name.